### PR TITLE
Added integration guide

### PR DESCRIPTION
**Pull request by OffcierCia**

### CODE REVIEW

File: docs/guides/stETH & wstETH Integration Paper.md [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+## Inheritance Structure
+
+The token contract inherits from the OpenZeppelin's `ERC20Permit`.
+
+## Other
+
+wstETH implements [EIP-2612 Permit](https://eips.ethereum.org/EIPS/eip-2612) standard for `secp256k1`-signed approvals.
+
+
+## Contracts Description Table
```
Comment (kadmil) [inline docs/guides/stETH & wstETH Integration Paper.md:368]: I think we can safely remove this section, tbh.

File: docs/guides/stETH & wstETH Integration Paper.md [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+
+#
+
+<img width="1125" src="https://github.com/OffcierCia/stETH-wstETH-Integration-paper/blob/main/01.png?raw=true" />
+
+
+Lido makes the stETH token balance track a balance of corresponding balance of beacon chain ether. A user’s balance of stETH tokens corresponds 1 to 1 to an amount of ether a user could receive if withdrawals were enabled and instant.
+
+The DAO selects node operators, which also validate transactions on the beacon chain and adds their addresses to the `NodeOperatorsRegistry` contract. Authorized node operators have to generate a set of keys for the validation and also provide them with the smart contract. As ether is received from users, it is distributed in chunks of 32 Ether between all active node operators. The staking pool contract contains a list of node operators, their keys, and the logic for distributing rewards between them.
```
Comment (kadmil) [inline docs/guides/stETH & wstETH Integration Paper.md:19]: Can't tell which contract is called "the staking pool contract" here, but the Node Operators list with the keys is in [NodeOperatorsRegistry](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/nos/NodeOperatorsRegistry.sol) contract, and the rewards distribution is in [Lido](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/Lido.sol)

Comment (OffcierCia) [reply docs/guides/stETH & wstETH Integration Paper.md]: + Fixed

File: docs/guides/stETH & wstETH Integration Paper.md (hunk 1 above)
Comment (OffcierCia) [reply docs/guides/stETH & wstETH Integration Paper.md]: + Fixed

**Pull request closed without merging**